src/content/en/updates/2018/12/badging-api.md

```diff
@@ -81,6 +81,11 @@ Examples of sites that may use this API include:
    running as an installed PWA (in your task bar or dock).
 3. Click the **Set** or **Clear** button to set or clear the badge from the app
    icon. You can also provide a number for the *Badge value*.
+   
+Note: While the Badging API requires an installed app with an icon that
+actually can be badged, we excplititly advise against making calls to
```

Comment:
"explicitly" is misspelled.

(I would drop this word anyway. The fact that you are explicitly stating this makes it explicit :) You don't need the word "explicit" in the document.)

Reply:
Dropped it like it's hot.